web-app/src/main/webapp/script.js

```diff
@@ -12,17 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function initMap() {
+  const map = new google.maps.Map(document.getElementById("map"), {
+    center: { lat: -34.397, lng: 150.644 },
+    zoom: 5,
+  });
+  const infoWindow = new google.maps.InfoWindow();
+  displayUserLocation(map, infoWindow);
+}
+
+function displayUserLocation(map, infoWindow) {
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(
+      function(position) {
+        let userPosition = {
+          lat: position.coords.latitude,
+          lng: position.coords.longitude,
```

Comment:
nit before Darren does: spell out lat and lng in full?

Reply:
hehe, yeah I don't think we can change this particular one because that's what's expected by the API.